python/Ganga/GPIDev/Adapters/IBackend.py

```diff
@@ -475,10 +472,6 @@ def master_updateMonitoringInformation(jobs):
                     temp_block = []
 
                 for this_block in monitorable_blocks:
-
-                    if monitoring_component and not monitoring_component.isEnabled(False) or not monitoring_component:
```

Comment:
The issue is that we want to be able to explicitly call the monitoring functions, even if the monitoring system is not running. In the case of the tests, the monitoring is not running and we call `master_updateMonitoringInformation` in a loop. This causes any jobs with subjobs to never be monitored.

What do you think about a global var in `ShutdownManager.py` which is set to `True` at the beginning of `_ganga_run_exitfuncs()`? Any code which wants to check if ganga is in the process of shutting down could then check that flag.